metadata-service/factories/src/main/java/com/linkedin/metadata/boot/UpgradeStep.java

```diff
@@ -0,0 +1,134 @@
+package com.linkedin.metadata.boot;
+
+import com.linkedin.common.AuditStamp;
+import com.linkedin.common.urn.Urn;
+import com.linkedin.data.DataMap;
+import com.linkedin.entity.EntityResponse;
+import com.linkedin.events.metadata.ChangeType;
+import com.linkedin.metadata.Constants;
+import com.linkedin.metadata.entity.EntityService;
+import com.linkedin.metadata.key.DataHubUpgradeKey;
+import com.linkedin.metadata.utils.EntityKeyUtils;
+import com.linkedin.metadata.utils.GenericRecordUtils;
+import com.linkedin.mxe.MetadataChangeProposal;
+import com.linkedin.upgrade.DataHubUpgradeRequest;
+import com.linkedin.upgrade.DataHubUpgradeResult;
+import java.net.URISyntaxException;
+import java.util.Collections;
+import javax.annotation.Nonnull;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+public abstract class UpgradeStep implements BootstrapStep {
+  private static final Integer SLEEP_SECONDS = 120;
+
+  protected final EntityService _entityService;
```

Comment:
why do an underscore on this one, but not the others?? let's keep it consistent, no?